server/views/account/signin.jade

```diff
@@ -5,7 +5,7 @@ block content
         .spacer
         a.btn.btn-lg.btn-block.btn-social.btn-primary(href='/email-signup')
             i.fa.fa-envelope
```

Comment:
I know we are using email, but should this still be an envelope? Would `.fa-user-o` make more sense?

Reply:
I am of the opinion that it is more associated with that of user profile.

Reply:
How about [fa-user-plus](http://fontawesome.io/icon/user-plus/)?